backend/src/serverless/integrations/usecases/hackerNews/getPostsByKeywords.ts

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios'
+import { IServiceOptions } from '../../../../services/IServiceOptions'
+import { EagleEyeResponses, EagleEyeInput } from '../../types/hackerNewsTypes'
+import { Logger } from '../../../../utils/logging'
+import { timeout } from '../../../../utils/timing'
+import { IS_DEV_ENV } from '../../../../config'
+import EagleEyeContentService from '../../../../services/eagleEyeContentService'
+
+async function getChannels(
```

Comment:
nit: Since this is a default export function name really doesn't matter but, for completeness let's call the function `getPostsByKeyword`